package.json

```diff
@@ -72,6 +74,7 @@
     "prettier": "^1.14.3",
     "standard-version": "^4.4.0",
     "webpack": "^4.20.2",
+    "webpack-cli": "^3.1.2",
```

Comment:
Installed for test and forgot to remove?

Reply:
yep! killing that with 🔥 now